hcconfig/interface.py

```diff
@@ -163,6 +175,52 @@ def do_version(self, arg=None):
         self.send('AT+VERSION?\r\n')
         print(self.get_response('VERSION:'))
 
+    def do_info(self, arg=None):
+        """
+        Get Device summary.
+        """
+
+        self.send('AT\r\n')
+        response = self.get_response().strip()
+        if not response == 'OK':
+            print('Please connect a module set to command mode')
+            return
+
+        self.send('AT+NAME?\r\n')
+        name = self.get_response('NAME:')
+        print(f'      Name: {name}')
+
+        self.send('AT+UART?\r\n')
+        uart = self.get_response('UART:').strip()
+        uart_parts = uart.split(',')
+        stop_bits = VALID['stopbits'][int(uart_parts[1])]
+        parity = VALID['parity'][int(uart_parts[2])]
+        print(f'  Baudrate: {uart_parts[0]}')
+        print(f' Stop Bits: {stop_bits}')
+        print(f'    Parity: {parity}')
+
+        self.send('AT+PSWD?\r\n')
+        password = self.get_response('')  # inconsistent response across modules
```

Comment:
I think you can just `self.get_response()`, right? (feels more natural to me without providing a parameter)

There are other similar calls that you can change if you want. Not that is very important to me...

Reply:
I felt is was a reminder that a search string was intended but left out. Your repo, your choice.

Reply:
I have made it consistent with the other calls. I tried to think of a way to give it a choice of matches but my eyes started to bleed.

Reply:
:joy: